torch/fx/graph.py

```diff
@@ -67,7 +67,7 @@ def map_arg(a: Argument, fn: Callable[[Node], Argument]) -> Argument:
 
 class Graph:
     def __init__(self):
-        self._nodes : List[Node] = []
+        self._nodes : List[Node] = [Node(self, 'return', 'return', 'return', (), {})]
```

Comment:
This is inconsistent with how placeholders are handled. A lot of the later complexity is related to the fact that the return node is inserted into the graph first. Why not insert it when `graph.output` is called (i.e. treat output like placeholder in that it inserts the node last).